docs/information/docker.md

```diff
@@ -30,12 +36,44 @@ docker run \
 * `--device=/dev/ttyACM0`: Location of adapter (e.g. CC2531)
 * `-v /run/udev:/run/udev:ro --privileged=true`: is optional, only required for autodetecting the port
 * Optional: in case your MQTT broker is running on `localhost` and is not within the same Docker network as the Zigbee2MQTT container also add `--network host \`.
+* Note: Running in "--privileged" mode is a security risk.
```

Comment:
Is this note really needed? Running as root is already a security risk, this might only confuse

Reply:
```suggestion
* Note: Running a docker container as root, especially in "--privileged" mode, is a security risk.
```

Reply:
I agree, I am not insisting on that line but maybe the root part should also be mentioned ;)

Reply:
What about removing the whole root section?

Reply:
If you mean the line: 
Fine for me.

If you mean the root paragraph: 
The root mode could be beneficial for auto-discovery mode for USB devices. However, I never tried auto-discovery, therefore, I cannot tell, how that feature works.

Reply:
I see, lets remove the line but also add a note that discovery only works when running as root (since it requires --privilliged).